packages/computed/spec/asyncBehaviors.ts

```diff
@@ -764,9 +764,9 @@ describe('Deferred', function () {
             // a given observable).
       var observable = koObservable()
 
-      observable.extend({deferred: true})
+      observable.extend({deferred: true});
       expect(function () {
-        observable.extend({deferred: false})
+        observable.extend(Object.assign({deferred: false}))
```

Comment:
This is a no-op. i.e. `Object.assign` isn't doing anything here.  It's harmless, but worth removing (either in this PR or the next)